huxley/www/js/stores/CurrentUserStore.js

```diff
@@ -33,6 +34,12 @@ class CurrentUserStore extends Store {
     return super.addListener(callback);
   }
 
+  updateSchool(schoolID, delta) {
+    const school = {...this._currentUser.school, ...delta};
+    ServerAPI.updateSchool(schoolID, delta);
+    this._currentUser.school = school;
```

Comment:
Remember not to mutate:
```js
const user = this._currentUser;
this._currentUser = {
  ...user,
  school: {...user.school, ...delta},
};
```
Adding the temp `user` binding here because it's shorter, lol.

It's a little clunky, but JS doesn't have builtin support for immutable objects yet.